### ISSUE TITLE
Formulae miscalculation 

### ISSUE THREAD

**Issue by deepakacl** [zhang0jhon/AttentionOCR#95]
**ZhangOjhon** your pre-trained model is gr8 !! for recognising characters generically. gr8 work it would be nice if you share the config file which includes the values used for the hyper parameters.  
I went through your code I just found out this formula discrepancy with the one in Bahadanus attention paper. 
1.) the hidden state derived from the lstm decoder layer should only be given to the softmax. But in the code the attention_feature, prev_wemb, hidden_layer is concatenated and then given to the softmax.
It would be helpful if you provide the explanation of why the formula is altered so and so. Thanks in advance.
